docs/developers/RELEASE.md

```diff
@@ -49,78 +53,51 @@ Thus, you'll need to tag the [reference](https://git-scm.com/book/en/v2/Git-Inte
 8. Make and merge a PR with these release notes before moving onto the next steps.
 
 
-## Tagging the new release candidate
+## tagging the new release candidate
 
 First we will generate a release candidate, which will contain the letters `rc`.
 Using release candidates allows us to test releases on PyPi without using up the actual
 release number.
 
 You should include a basic message with the tag `"Version X.Y.Zrc1"`:
```

Comment:
can you comment on whether this is this still necessary, if we're removing the `-m "Version...` bit from the command?

Reply:
good catch! I forgot to remove that, oops!